bridges/deployments/networks/OpenEthereum.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial AS builder
```

Comment:
@tomusdrw is this image relevant to Polkadot?
Also, Xenial (16.04) is out of standard support since April 2021, our containers policy is not happy.

Reply:
No I don't think so. If possible please avoid touching `bridges` code in this repo, it's only a clone of the `bridges` repository and the changes are better done there first and then synced here to avoid conflicts.

Is it caught by some blanket CI checks? Can we ignore `bridges` sub-dir from any checks of this kind? In the future the plan is to integrate the bridges code more closely with Polkadot/Substrate repositories, but for now we would rather keep focus on making sure the code works correctly in our repo.

Reply:
yeah, a more general question was "what the entire bridges repo does here?" and now I get it, thanks. I'll make a change in bridges then.

Reply:
It's a `git subtree`, because we want to use some of the crates from bridges repo, but keep working on the separately. Other option is referencing crates using git repo, but that makes it cumbersome in case of compilation errors (i.e. companion PR system).